conf/base.config

```diff
@@ -1,91 +1,88 @@
-  }
-  withName:plot_entity_binding_ratios {
-    cpus = { check_max( 1 * task.attempt, 'cpus' ) }
-    memory = { check_max( 60.GB * task.attempt, 'memory' ) }
-    time = { check_max( 1.h * task.attempt, 'time' ) }
-    cache = 'lenient'
-  }
-  withName:get_software_versions {
-    cache = false
-  }
+
+    errorStrategy = { task.exitStatus in [143,137,104,134,139] ? 'retry' : 'finish' }
+    maxRetries = 3
+    maxErrors = '-1'
+
+    // Process-specific resource requirements
+    // NOTE - Please try and re-use the labels below as much as possible.
+    //        These labels are used and recognised by default in DSL2 files hosted on nf-core/modules.
+    //        If possible, it would be nice to keep the same label naming convention when
+    //        adding in your local modules too.
+    // See https://www.nextflow.io/docs/latest/config.html#config-process-selectors
+    withLabel:process_low {
+        cpus   = { check_max( 2     * task.attempt, 'cpus'    ) }
+        memory = { check_max( 12.GB * task.attempt, 'memory'  ) }
+        time   = { check_max( 4.h   * task.attempt, 'time'    ) }
+    }
+    withLabel:process_medium {
+        cpus   = { check_max( 6     * task.attempt, 'cpus'    ) }
+        memory = { check_max( 36.GB * task.attempt, 'memory'  ) }
+        time   = { check_max( 8.h   * task.attempt, 'time'    ) }
+    }
+    withLabel:process_high {
+        cpus   = { check_max( 12    * task.attempt, 'cpus'    ) }
+        memory = { check_max( 72.GB * task.attempt, 'memory'  ) }
+        time   = { check_max( 16.h  * task.attempt, 'time'    ) }
+    }
+    withLabel:process_long {
+        time   = { check_max( 20.h  * task.attempt, 'time'    ) }
+    }
+    withLabel:process_high_memory {
+        memory = { check_max( 200.GB * task.attempt, 'memory' ) }
+    }
+    withLabel:error_ignore {
+        errorStrategy = 'ignore'
+    }
+    withLabel:error_retry {
+        errorStrategy = 'retry'
+        maxRetries    = 2
+    }
+    withName:CUSTOM_DUMPSOFTWAREVERSIONS {
+        cache = false
+    }
+    withName:GENERATE_PEPTIDES {
+        cpus = { check_max( 1 * task.attempt, 'cpus' ) }
+        cache = 'lenient'
+    }
+    withName:COLLECT_STATS {
+        cpus = { check_max( 1 * task.attempt, 'cpus' ) }
+        cache = 'lenient'
+    }
+    withName:SPLIT_PRED_TASKS {
+        cpus = { check_max( 1 * task.attempt, 'cpus' ) }
+        cache = 'lenient'
+    }
+    withName:PREDICT_EPITOPES {
+        cache = 'lenient'
+    }
+    withName:MERGE_PREDICTIONS_BUFFER {
+        memory = { check_max( 60.GB * task.attempt, 'memory' ) }
+        cache = 'lenient'
+    }
+    withName:MERGE_PREDICTIONS {
+        cache = 'lenient'
+    }
+    withName:PLOT_SCORE_DISTRIBUTION {
+        memory = { check_max( 60.GB * task.attempt, 'memory' ) }
+        cache = 'lenient'
+    }
+    withName:PLOT_ENTITY_BINDING_RATIOS {
+        memory = { check_max( 60.GB * task.attempt, 'memory' ) }
+        cache = 'lenient'
+    }
```

Comment:
If possible these should be replaced by the `withLabel` definitions.

Reply:
Creating a new label 'cache_lenient', leaving withName statements if no suitable label available